kolibri/plugins/learn/assets/src/views/TopicsPage/index.vue

```diff
@@ -693,7 +693,7 @@
 <style lang="scss" scoped>
 
   $header-height: 324px;
-  $toolbar-height: 70px;
+  $toolbar-height: 50px;
```

Comment:
This is causing the following regression on non-small screens:

![spacing](https://user-images.githubusercontent.com/13509191/211309055-a0c7af28-e6a0-4abe-afea-06ca31c53b85.png)

Needs to be fixed.

Reply:
@MisRob Thanks for letting me know. I am taking a look at it. I will revert back